src/core_plugins/kibana/public/visualize/listing/visualize_listing.html

```diff
@@ -32,26 +32,11 @@
 
       <div class="kuiToolBarSection">
         <!-- Bulk delete button -->
-        <button
-          class="kuiButton kuiButton--danger"
-          aria-label="Delete selected objects"
-          ng-if="listingController.getSelectedItemsCount() > 0"
-          ng-click="listingController.deleteSelectedItems()"
-          tooltip="Delete selected visualizations"
-        >
-          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-trash"></span>
-        </button>
+        <react-component name="TrashButtonComponent" props="trashButtonProps"></react-component>
```

Comment:
How cool, I didn't know you could use ng-if on the ngreact element.  I wonder if that will still work with the reactDirective syntax.  I will try it out!